gammapy/image/tests/test_radial_profile.py

```diff
@@ -0,0 +1,26 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from gammapy.image import SkyImage
```

Comment:
Please put the gammapy imports together after the astropy imports.
And use "explicit relative imports", not "absolute imports" for Gammapy (see any other file as an example).